pipeline_dp/private_spark.py

```diff
@@ -0,0 +1,60 @@
+from pyspark import RDD
+from typing import Callable
+
+import pipeline_dp
+from pipeline_dp import aggregate_params, budget_accounting
+
+
+class PrivateRDD:
+    """ A Spark RDD counterpart.
+
+    PrivateRDD guarantees that only anonymized data
+    within the specified privacy budget can be extracted from it through its API.
+    """
+
+    def __init__(self, rdd, budget_accountant, privacy_id_extractor):
+        self._rdd = rdd
+        self._budget_accountant = budget_accountant
+        self._privacy_id_extractor = privacy_id_extractor
+
+    def sum(
+        self,
+        sum_params: aggregate_params.SumParams,
+        noise_kind: aggregate_params.NoiseKind = pipeline_dp.NoiseKind.GAUSSIAN
```

Comment:
Please add `noise_kind` to `sum_params` (the same way as in Aggregate params). No need to have default value